Code/MinimalLib/common.h

```diff
@@ -448,6 +449,30 @@ std::string process_rxn_details(
   return "";
 }
 
+std::string molblock_helper(RWMol &mol, const char *details_json, bool forceV3000) {
+  bool includeStereo = true;
+  bool kekulize = true;
+  bool useMolBlockWedging = false;
+  bool addChiralHs = false;
+  if (details_json && strlen(details_json)) {
+    boost::property_tree::ptree pt;
+    std::istringstream ss;
+    ss.str(details_json);
+    boost::property_tree::read_json(ss, pt);
+    LPT_OPT_GET(includeStereo);
+    LPT_OPT_GET(kekulize);
+    LPT_OPT_GET(useMolBlockWedging);
+    LPT_OPT_GET(addChiralHs);
+  }
+  if (useMolBlockWedging) {
```

Comment:
We should really create a version of `MolToMolBlock()` that (optionally) takes care of stuff like `reapplyMolBlockWedging()` and `addChiralHs()`

Reply:
not for this PR, of course